frontend/src/app/register/register.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { RegisterRequestModel } from 'app/auth.models';
+import { AuthService } from '../auth.service';
+import { Observable } from 'rxjs/Observable';
+import { RedirectComponent } from 'app/redirect.component';
+import { Params } from '@angular/router';
+
+@Component({
+  selector: 'volontulo-register',
+  templateUrl: './register.component.html',
+})
+export class RegisterComponent implements OnInit {
+  registerModel: RegisterRequestModel = {
+    email: '',
+    password: '',
+  };
+  ACCEPT_TERMS = 'Wyrażam zgodę na przetwarzanie moich danych osobowych';
+  registrationSuccessful = false;
+  userIsAuthenticated = false;
+
+  constructor(private authService: AuthService,
+  ) {
+  }
+
+  register(): void {
+    this.registrationSuccessful = false;
+    this.userIsAuthenticated = false;
+    this.authService.register(this.registerModel.email, this.registerModel.password)
+      .subscribe(rsp => {
+        if (rsp.status === 201) {
+          this.registrationSuccessful = true;
```

Comment:
After successful registration, this component should probably redirect to homepage or login page or, if we want the message to be displayed on this page, resetting the form may be a good idea.